tests/meta/orders/test_meta_orders_status.py

```diff
@@ -697,3 +697,46 @@ def test_store_existing_single_sample_from_trio(
     assert len(new_families[0].links) == 1
     assert not new_families[0].links[0].mother
     assert not new_families[0].links[0].father
+
+
+def test_store_rerun_case(orders_api, base_store, mip_status_data, ticket: str):
+    # GIVEN a basic store with no samples or nothing in it + scout order
+    assert base_store.samples().first() is None
+    assert base_store.families().first() is None
+
+    submitter: MipDnaSubmitter = MipDnaSubmitter(lims=orders_api.lims, status=orders_api.status)
+
+    # WHEN storing the order
+    new_families = submitter.store_items_in_status(
+        customer=mip_status_data["customer"],
+        order=mip_status_data["order"],
+        ordered=dt.datetime.now(),
+        ticket=ticket,
+        items=mip_status_data["families"],
+    )
+
+    base_store.close()
+    new_families = base_store.families().all()
+
+    # Save internal id
+    stored_cases_internal_ids = dict(
+        [(family["name"], family["internal_id"]) for family in new_families]
+    )
+    for family in mip_status_data["families"]:
+        family["internal_id"] = stored_cases_internal_ids[family["name"]]
+
+    new_ticket = "234567"
```

Comment:
I need I new ticket number that is *not* '123456', I only found fixture_ticket_number which returns "123456" so that wouldn't work. Is there anything I'm missing?

Reply:
Does the ticket number really matter? Are you not testing that a str gets appended (even it is the same str repeated). However, I agree that it is more logical with another ticket number. Perhaps this is needed in more situations - so you can create another ticket fixture and use that instead. It is good to have a habit of (almost) never hard coding.

Reply:
You are right that and the ticket number doesn't really matter. For now I'll use the existing fixture and we can add another fixture in the future if needed